lib/yaml_resolver.py

```diff
@@ -266,11 +266,11 @@ def _resolve(obj, obj_path, obj_file_path, doc_obj, arch_root, do_checks):
 
       if "$parent_of" in ref_obj:
         if isinstance(ref_obj["$parent_of"], list):
-          ref_obj["$parent_of"].append(f"{obj_file_path}#/{"/".join(obj_path)}")
+          ref_obj["$parent_of"].append(f'{obj_file_path}#/{"/".join(obj_path)}')
```

Comment:
For this and the next two, (this is a nit, but) I would've just put single quotes around the `/` character.